sdks/python/apache_beam/io/tfrecordio_test.py

```diff
@@ -24,21 +24,22 @@
 import pickle
 import random
 import re
-import shutil
-import tempfile
 import unittest
+from unittest import TestCase
```

Comment:
<!--new_thread; commit:d4254d9fe817a78ccd0f7704c87bc89f903add8c; resolved:0-->
For consistency, I would stick to either
  import unittest
or
  from unittest import foo

Reply:
Done.